modules/acm/main.tf

```diff
@@ -14,6 +14,19 @@
  * limitations under the License.
  */
 
+module "enable_acm" {
+  source  = "terraform-google-modules/gcloud/google"
+  version = "~> 2.0"
+
+  platform = "linux"
```

Comment:
We should also expose `service_account_key_file ` so users can pass in credentials to auth gcloud incase they are running in an env without ADC.

Reply:
Good catch. Added.